examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -6,19 +6,22 @@
 import { CollaborativeInput } from "@fluid-experimental/react-inputs";
 
 import React, { useEffect, useRef, useState } from "react";
+import { useSetRecoilState } from "recoil";
 
 import type { ITask, ITaskList } from "../model-interface";
+import { unresolvedConflicts } from "./recoilState";
 
 interface ITaskRowProps {
 	readonly task: ITask;
 	readonly deleteTask: () => void;
+	readonly trackChanges: () => void;
```

Comment:
Can we document what this param does? The other two probably should probably also be documented, but that's outside the scope of this PR.